jobserv/models.py

```diff
@@ -460,29 +461,35 @@ def pop_queued(worker):
         rows = cursor.execute('''
             SELECT
               runs.id, runs.build_id, runs._status,
-              projects.id, projects.synchronous_builds
+              projects.id, projects.synchronous_builds, runs.host_tag
             FROM runs
             JOIN builds on builds.id = runs.build_id
             JOIN projects on projects.id = builds.proj_id
             WHERE
                 runs._status in (1, 2)
-              AND ({tags})
               ORDER BY
                 runs._status DESC, runs.queue_priority DESC,
                 runs.build_id ASC, runs.id ASC
-            '''.format(tags=tags))
+            ''')
 
+        tags = [worker.name] + [x.strip() for x in worker.host_tags.split(',')]
         # By ordering the query above by Run._status, we'll get the active
         # runs first so that we can build up this list of build ids that are
         # active for synchronous projects runs with these build ids are okay
         # to schedule
         sync_projects = {}
         okay_sync_builds = {}
-        for (run_id, build_id, status, proj_id, sync) in cursor.fetchall():
+        for (run_id, build_id, status, proj_id, sync, tag) in cursor.fetchall():
+            print(run_id, build_id, status, tag)
```

Comment:
Did you want a print here?

Reply:
nope - left over debug. force-push should correct that.